lib/shoryuken/manager.rb

```diff
@@ -78,6 +78,7 @@ def processor_done(queue, processor)
     def processor_died(processor, reason)
       watchdog("Manager#processor_died died") do
         logger.error { "Process died, reason: #{reason}" unless reason.to_s.empty? }
+        reason.backtrace.each { |bt| logger.error { bt } } unless reason.backtrace.nil?
```

Comment:
```ruby
2016-11-02T22:38:50Z 21002 TID-ow6zrttks ERROR: Manager#processor_died died
2016-11-02T22:38:50Z 21002 TID-ow6zrttks ERROR: undefined method `backtrace' for nil:NilClass
```

@barisbalic just got `reason = nil` ^ 

Actually, I'm having a hard time to find the celluloid documentation on `trap_exit` to understand what `reason` is, so for now, I would say it's better to not add it, as it isn't reliable to be present.